docs/telemetryData.md

```diff
@@ -121,87 +124,133 @@ As new features and pages roll out, percentages  will adjust.
 
 |Project|Percentage|
 |:---|:---:|
-|Navigation View|56.4%|
-|MenuBar|16%|
-|Ribbon|14.6%|
-|Blank|13%|
+|Navigation View|65.3%|
+|Ribbon|12.2%|
+|MenuBar|11.3%|
+|Blank|11.1%|
 
 ### Framework
 
 |Framework Type|Percentage|
 |:---|:---:|
-|CodeBehind|36.1%|
-|MVVM Basic|27.4%|
-|Prism|22.1%|
-|MVVM Light|14.2%|
-|MVVMToolkit|0.3%|
+|MVVMToolkit|55.1%|
+|CodeBehind|19.8%|
+|Prism|15.3%|
+|MVVM Basic|5.6%|
+|MVVM Light|4.2%|
 
 ### Pages
 
 |Pages|Percentage|
 |:---|:---:|
-|Blank|40.6%|
-|Settings|17.8%|
-|Data Grid|11.7%|
-|Content Grid|10.7%|
-|MasterDetail|8.9%|
-|Web View|4.3%|
-|Master Detail|3.8%|
-|XAML Island|2%|
-|ListDetails|0.1%|
+|Blank|41.1%|
+|Settings|17.7%|
+|Data Grid|12.4%|

```

Comment:
Reviewing all ListDetail and MasterDetail and summing them up

|ListDetails|9%|
|List Details|1.6%|
|MasterDetail|1.1%|
|Master Detail|0.6%|


```suggestion
|Data Grid|12.4%|
|ListDetail|12.3%|
```